docker/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM golang:1.17 as builder
+RUN apt update && \
+    apt install make bash
+ENV HOME /celestia-node
+COPY / ${HOME}
+WORKDIR ${HOME}
+RUN make build 
+
+FROM ubuntu
```

Comment:
out of curiosity, is it possible to have alpine linux instead of ubuntu? 🧐 
imho making docker image with smaller footprint will be great

Reply:
I tried using Alpine but there's something not working in it, probably related to musl libc and some cgo thing. I didn't want to track it down so I punted to Ubuntu.

Reply:
what would be the difference in image size if you were to download what is needed in an alpine image vs using ubuntu?